README.md

```diff
@@ -1,8 +1,8 @@
-# App Store for DHIS 2
+# App Hub for DHIS 2
 
-Visit the live [DHIS 2 app store](https://play.dhis2.org/appstore/)
+Visit the live [DHIS 2 app hub](https://play.dhis2.org/apphub/)
```

Comment:
The URL should be https://apps.dhis2.org.

Reply:
Resolved but not changed?

Reply:
sorry, changed an url the same way on another place... now also updated the readme